pkg/plugins/s3/spec.go

```diff
@@ -14,17 +14,85 @@
 
 package s3
 
-import "github.com/conduitio/conduit/pkg/plugins"
+import (
+	"github.com/conduitio/conduit/pkg/plugin/sdk"
+	"github.com/conduitio/conduit/pkg/plugins/s3/config"
+	"github.com/conduitio/conduit/pkg/plugins/s3/destination"
+	"github.com/conduitio/conduit/pkg/plugins/s3/source"
+)
 
 type Spec struct{}
 
-// Specify returns the Plugin's Specification
-func (s Spec) Specify() (plugins.Specification, error) {
-	return plugins.Specification{
-		Summary:           "An S3 source and destination plugin for Conduit, written in Go.",
-		Version:           "v0.0.1",
-		Author:            "Meroxa, Inc.",
-		DestinationParams: map[string]plugins.Parameter{},
-		SourceParams:      map[string]plugins.Parameter{},
-	}, nil
+// Specification returns the Plugin's Specification.
+func Specification() sdk.Specification {
+	return sdk.Specification{
+		Name:    "s3",
+		Summary: "An S3 source and destination plugin for Conduit, written in Go.",
+		Version: "v0.0.1",
+		Author:  "Meroxa, Inc.",
+		DestinationParams: map[string]sdk.Parameter{
+			config.ConfigKeyAWSAccessKeyID: {
+				Default:     "",
+				Required:    true,
+				Description: "AWS access key id.",
+			},
+			config.ConfigKeyAWSSecretAccessKey: {
+				Default:     "",
+				Required:    true,
+				Description: "AWS secret access key.",
+			},
+			config.ConfigKeyAWSRegion: {
+				Default:     "",
+				Required:    true,
+				Description: "the AWS S3 bucket region.",
+			},
+			config.ConfigKeyAWSBucket: {
+				Default:     "",
+				Required:    true,
+				Description: "the AWS S3 bucket name.",
+			},
+			destination.ConfigKeyBufferSize: {
+				Default:     "1000",
+				Required:    false,
+				Description: "the buffer size {when full, the files will be written to destination}, max is \"100000\".",
+			},
+			destination.ConfigKeyFormat: {
+				Default:     "",
+				Required:    false,
+				Description: "the destination format, either \"json\" or \"parquet\".",
```

Comment:
Protip: use backticks when you need to use quotes in a string, then you don't have to escape them:
```suggestion
				Description: `the destination format, either "json" or "parquet".`,
```